### PR TITLE
OnlyHit

### PR DESCRIPTION
**Pull request by a6doff**
Adding onlyhit.us
Proof of it working:
image

### CODE REVIEW

**Review by a6doff — COMMENTED** [review]
Changed some stuff

File: websites/O/OnlyHit/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "a6d",
+    "id": "523890815291097129"
+  },
+  "url": "onlyhit.us",
+  "description": {
+    "en": "Extension to show what you are listening to on OnlyHit"
+  },
+  "service": "OnlyHit",
+  "version": "1.0.0",
+  "logo": "https://api.onlyhit.us/logo.png",
```
Comment (Alanexei) [inline websites/O/OnlyHit/dist/metadata.json:13]: Publish this to imgur.

**Review by Alanexei — CHANGES_REQUESTED** [review]

**Comment by a6doff** [comment]
Uploaded image to imgur

File: websites/O/OnlyHit/dist/metadata.json (hunk 1 above)
Comment (a6doff) [reply websites/O/OnlyHit/dist/metadata.json]: done

**Comment by a6doff** [comment]
All edited

File: websites/O/OnlyHit/presence.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      largeImageKey: `${logo}`,
+      smallImageKey: `${status}`,
+      smallImageText: `Viewing: ${pageName}`
+    };
+  
+    presenceData.details = `${title}`;
+    presenceData.state = `${radio} on OnlyHit.us`;
+
+    let lastTimeStart = Math.floor(Date.now() / 1000);
+    lastTimeStart = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/O/OnlyHit/presence.ts:30]: this doesnt seem needed
(twice the same thing)

File: websites/O/OnlyHit/presence.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+  const presence = new Presence({
+    clientId: "666412985513672715"
+  });
+  
+  presence.on("UpdateData", () => {
+    const title = `${document.querySelector(".artist").textContent} - ${document.querySelector(".title").textContent} `,
+          radio = document.querySelector(".stream-name").textContent;
```
Comment (Bas950) [inline websites/O/OnlyHit/presence.ts:7]: move the consts down to the pageName one and then combine the consts

File: websites/O/OnlyHit/presence.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    let logo = 'logo_onlyhit';
+    const status = 'play';
+    if (document.querySelector(".stream-name").textContent == 'OnlyHit Gold') {
```
Comment (Bas950) [inline websites/O/OnlyHit/presence.ts:10]: move the consts down to the pageName one and then combine the consts

File: websites/O/OnlyHit/presence.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    presenceData.details = `${title}`;
+    presenceData.state = `${radio} on OnlyHit.us`;
+
+    let lastTimeStart = Math.floor(Date.now() / 1000);
+    lastTimeStart = Math.floor(Date.now() / 1000);
+    presenceData.startTimestamp = lastTimeStart;
+  
+    presence.setActivity(presenceData);
+    presence.setTrayTitle();
+  });
```
Comment (Bas950) [inline websites/O/OnlyHit/presence.ts:35]: set traytitle not needed

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by a6doff** [comment]
(Updated)

**Review by Alanexei — APPROVED** [review]

**Pull request merged**